py/objtype.c

```diff
@@ -591,6 +597,11 @@ STATIC void mp_obj_instance_load_attr(mp_obj_t self_in, qstr attr, mp_obj_t *des
     mp_obj_class_lookup(&lookup, self->base.type);
     mp_obj_t member = dest[0];
     if (member != MP_OBJ_NULL) {
+        if (!(self->base.type->flags & TYPE_FLAG_HAS_SPECIAL_ACCESSORS)) {
```

Comment:
Why this overhead code isn't wrapped with `#if ENABLE_SPECIAL_ACCESSORS`?

Reply:
I checked this (wrapping it in `ENABLE_SPECIAL_ACCESSORS`) and it doesn't do anything, it seems the compiler is smart enough to eliminate it anyway.  That's because if there are no special accessors none of the code below exists, and the statement after this `if` is immediately a `return`.